src/bidiMapper/CommandProcessor.ts

```diff
@@ -311,7 +311,7 @@ export class CommandProcessor extends EventEmitter<CommandProcessorEventsMap> {
       // Session domain
       // keep-sorted start block=yes
       case 'session.new':
-        return await this.#sessionProcessor.create(command.params);
+        return await this.#sessionProcessor.new(command.params);
```

Comment:
can it be considered a reserved keyword by some tools?

Reply:
it seems to be a valid method name